tests/Unit/CMSiteConfiguration.tests.ps1

```diff
@@ -0,0 +1,1594 @@
+                    $result.RequireSigning                                    | Should -Be -ExpectedValue $true
+                    $result.RequireSha256                                     | Should -Be -ExpectedValue $true
+                    $result.UseEncryption                                     | Should -Be -ExpectedValue $true
+                    $result.MaximumConcurrentSendingForAllSite                | Should -Be -ExpectedValue 6
+                    $result.MaximumConcurrentSendingForPerSite                | Should -Be -ExpectedValue 3
+                    $result.RetryNumberForConcurrentSending                   | Should -Be -ExpectedValue 3
+                    $result.ConcurrentSendingDelayBeforeRetryingMins          | Should -Be -ExpectedValue 2
+                    $result.EnableLowFreeSpaceAlert                           | Should -Be -ExpectedValue $false
+                    $result.FreeSpaceThresholdWarningGB                       | Should -Be -ExpectedValue $null
+                    $result.FreeSpaceThresholdCriticalGB                      | Should -Be -ExpectedValue $null
+                    $result.ThresholdOfSelectCollectionByDefault              | Should -Be -ExpectedValue 100
+                    $result.ThresholdOfSelectCollectionMax                    | Should -Be -ExpectedValue 200
+                    $result.SiteSystemCollectionBehavior                      | Should -Be -ExpectedValue 'Block'
+                    $result.SiteType                                          | Should -Be -ExpectedValue 'Primary'
+                    $result.EnableWakeOnLan                                   | Should -Be -ExpectedValue $false
+                    $result.WakeOnLanTransmissionMethodType                   | Should -Be -ExpectedValue $null
+                    $result.RetryNumberOfSendingWakeupPacketTransmission      | Should -Be -ExpectedValue $null
+                    $result.SendingWakeupPacketTransmissionDelayMins          | Should -Be -ExpectedValue $null
+                    $result.MaximumNumberOfSendingWakeupPacketBeforePausing   | Should -Be -ExpectedValue $null
+                    $result.SendingWakeupPacketBeforePausingWaitSec           | Should -Be -ExpectedValue $null
+                    $result.ThreadNumberOfSendingWakeupPacket                 | Should -Be -ExpectedValue $null
+                    $result.SendingWakeupPacketTransmissionOffsetMins         | Should -Be -ExpectedValue 0
+                    $result.ClientCertificateCustomStoreName                  | Should -Be -ExpectedValue 'SMSStore'
+                    $result.TakeActionForMultipleCertificateMatchCriteria     | Should -Be -ExpectedValue 'FailSelectionAndSendErrorMessage'
+                    $result.ClientCertificateSelectionCriteriaType            | Should -Be -ExpectedValue 'ClientAuthentication'
+                    $result.ClientCertificateSelectionCriteriaValue           | Should -Be -ExpectedValue ''
```

Comment:
in get doesn't this default to Personal?

Reply:
That's the ClientCertificateCustomStoreName parameter you're thinking of. It does default to personal. The test for that is captured on line 913.